src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -197,7 +197,7 @@ void DlgPrefLibrary::slotResetToDefaults() {
             kHistoryTrackDuplicateDistanceDefault);
     spinbox_history_min_tracks_to_keep->setValue(1);
     checkBox_SyncTrackMetadata->setChecked(false);
-    checkBox_SeratoMetadataExport->setChecked(false);
+    checkBox_SeratoMetadataExport->setChecked(true);
```

Comment:
Serato metadata export should not be enabled by default. Please revert this unrelated change.